core/src/test/scala/com/metamx/tranquility/test/TranquilizerTest.scala

```diff
@@ -309,7 +309,7 @@ class TranquilizerTest extends FunSuite with Matchers with Logging
 
 object TranquilizerTest
 {
-  val MomentsSoDear = Seq(525600 * 60000)
+  val MomentsSoDear = Seq(-1)
```

Comment:
Could you make this `Seq(525600 * 60000, -1)` (assuming this does not cause problems with the tests)?

Reason being, the tests for "long lingering" should work with both a big value and with -1.